time stamp
float32 left #left_motor encoder values
float32 right #right_motor encoder values
